### PR TITLE
fix: add sibilant support

### PR DESCRIPTION
**Pull request by Beraliv**
- [x] add examples
- [x] fix tests
- [x] add more examples
- [x] reoder rules (move less important to the end, move more descriptive to the top)

### CODE REVIEW

**Comment by Beraliv** [comment]
UPDATE: found some interesting cities, need to update PR

File: src/rules/cityRules.js [hunk 1]
```diff
@@ -60,18 +55,48 @@ export const customInflections: DeclensionRuleSetT = {
   suffixes: [
     {
       gender: 'androgynous',
-      test: ['ий', 'ое'], // Волжский, Раменское
-      mods: ['--ого', '--ому', '', '-м', '--ом'],
+      test: ['чёк', 'чек'], // Волочёк, Чернечек
```
Comment (Beraliv) [inline src/rules/cityRules.js]: is it essential to use `male` here and below?

File: src/rules/cityRules.js [hunk 2]
```diff
@@ -60,18 +55,48 @@ export const customInflections: DeclensionRuleSetT = {
...
+      mods: ['-а', '-у', '', 'м', ''],
+    },
+    {
+      gender: 'androgynous',
+      test: ['чье'], // Щучье
+      mods: ['-я', '-ю', '', 'м', ''],
+    },
+    {
+      gender: 'androgynous',
+      test: ['чь'], // Холмечь
```
Comment (Beraliv) [inline src/rules/cityRules.js]: is it essential to use `female` here and below?

**Comment by nodkz** [comment]
Thanks for PR!
🍻

**Pull request merged**